client/my-sites/media-library/content.jsx

```diff
@@ -163,6 +181,32 @@ export default React.createClass( {
 		return createFragment( notices );
 	},
 
+	countStorageErrorImpression() {
+		analytics.tracks.recordEvent( 'calypso_upgrade_nudge_impression', {
+			cta_name: 'plan-media-storage-error'
+		} );
+	},
+
+	renderNoticeAction( errorType ) {
+		if ( STORAGE_ERRORS.indexOf( errorType ) === -1 ) {
+			return null;
+		}
+		return (
+			<NoticeAction href="#" onClick={ this.navigateToPlans } onMount={ this.countStorageErrorImpression } >
```

Comment:
Aside: Kinda wish we would render a `<button />` from `<NoticeAction />` if we didn't want to specify a `href`.